contributing-guides/git-terminal.md

```diff
@@ -29,3 +31,32 @@ Typically a pull request will include changes in a single file.
 [pr-howto]: ../CONTRIBUTING.md#submitting-a-pull-request
 [commit-msg]: ../CONTRIBUTING.md#commit-message
 [mass-changes]: https://github.com/tldr-pages/tldr/pulls?&q=is:pr+is:merged+label:"mass+changes"
+
+# Updating your fork
+
+Forks of GitHub repositories aren't updated automatically. In order to keep your fork of the repository up-to-date with the latest changes and avoid merge conflicts, you should update it regularly.
+
+There are two ways to update your fork.
```

Comment:
I think it'd be better to somehow make clear the list is not exhaustive?
"There are two ways" literally states that there are a fixed number of methods, but there are actually far more than two.
The fact that we're just documented two common/convenient ways should be made clear in my opinion.

```suggestion
There are primarily two ways to update your fork.
```

```suggestion
Two ways that you can update your fork.
```

Reply:
But those are the only practical methods to update it as far as I know. So for a beginner the sentense is easier to read and someone who would want to do it another way prabably doesn't read the guide in the first place.

Reply:
I'll concede and resolve the conversation.

Reply:
I'll concede and resolve the conversation.